src/content/configuration/configuration-languages.md

```diff
@@ -135,14 +135,14 @@ In the example below JSX (React JavaScript Markup) and Babel are used to create
 First install the necessary dependencies:
 
 ``` bash
-npm install --save-dev babel-register jsxobj babel-preset-es2015
+npm install --save-dev babel-register jsxobj @babel/preset-env
 ```
 
 __.babelrc__
 
 ``` json
 {
-  "presets": [ "es2015" ]
+  "presets": [ "env" ]
```

Comment:
Has anyone actually tried this example? I don't think it will work as advertised: https://github.com/developit/jsxobj/blob/master/.babelrc

Reply:
What do you mean?

Reply:
It's not enough to set the preset for `jsxobj` to work, you also need to setup `transform-react-jsx` plugin correctly. See how they set the `pragma` setting to a different function in the linked `.babelrc`?

Reply:
I'm not talking about the lib, I'm talking about JSX in the config: https://github.com/webpack/webpack.js.org/blame/master/src/content/configuration/configuration-languages.md#L161-L176

Reply:
Never used it, but I think it is handle when you name your webpack config __webpack.config.babel.js__.

I will try it later to see if that guide actually works